pom.xml

```diff
@@ -93,4 +93,52 @@
     </dependency>
 
   </dependencies>
+
+  <build>
+    <plugins>
+
+      <!-- Override parent pom to suppress enforcing dependency convergence -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.3.1</version>
+        <executions>
+          <execution>
+            <id>enforce-versions</id>
+            <phase>initialize</phase>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration combine.self="override">
+              <rules>
+                <requireMavenVersion>
+                  <version>3.0</version>
+                </requireMavenVersion>
+                <requireJavaVersion>
+                  <version>${java.minimum.version}</version>
+                </requireJavaVersion>
+              </rules>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- Override parent pom to suppress treating warnings as errors -->
+      <plugin>
```

Comment:
Depending on these you might just want to fix them -- they are the compiler telling you that things could be wrong.

Reply:
That's a good point. I started going down this path, but there are a lot of changes needed to clear up the warnings. I'm concerned that if we diverge too much from the original repo that it'll be harder to keep it up-to-date.